goinitrd.rcS

```diff
@@ -0,0 +1,18 @@
+#!/bin/sh
```

Comment:
the standard u-root init does all this. If it's missing something lmk and I'll fix it

Reply:
ah but not the ip stuff. The way we do that in u-root is you provide a program, typically called uinit.go, which does special things you need. We try to avoid shell scripts in u-root to avoid a shell dependency, but I'm happy to do whatever you wish. You can all name this file /uinit and it will be run by the u-root init.

Reply:
btw, this is really nice to see.